### PR TITLE
refactor: aws-for-fluent to use helm-addon module

### PR DESCRIPTION
**Pull request by Zvikan**
### More

- [ ] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)
- [ ] Yes, I have added a new example under [examples](https://github.com/aws-samples/aws-eks-accelerator-for-terraform/tree/main/examples) to support my PR
- [ ] Yes, I have created another PR for add-ons under [add-ons](https://github.com/aws-samples/ssp-eks-add-ons) repo (if applicable)
- [ ] Yes, I have updated the [docs](https://github.com/aws-samples/aws-eks-accelerator-for-terraform/tree/main/docs) for this feature
- [ ] Yes, I ran `pre-commit run -a` with this PR

**Note**: Not all the PRs required examples and docs except a new pattern or add-on added.

### For Moderators
- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: modules/kubernetes-addons/aws-for-fluentbit/locals.tf [hunk 1]
```diff
@@ -67,4 +41,14 @@ locals {
     logGroupName       = aws_cloudwatch_log_group.aws_for_fluent_bit.name
     serviceAccountName = local.service_account_name
   }
+
+    irsa_config = {
```
Comment (askulkarni2) [inline modules/kubernetes-addons/aws-for-fluentbit/locals.tf:45]: extra indentation

**Review by askulkarni2 — COMMENTED** [review]
Looks good to me. Minor comment.

File: modules/kubernetes-addons/aws-for-fluentbit/locals.tf [hunk 2]
```diff
@@ -14,41 +15,14 @@ locals {
...
-    disable_openapi_validation = false
-    dependency_update          = false
-    replace                    = false
-    postrender                 = ""
+    name        = local.name
+    chart       = local.name
+    repository  = "https://aws.github.io/eks-charts"
+    version     = "0.1.11"
+    namespace   = "logging"
+    timeout     = "1200"
```
Comment (kcoleman731) [inline modules/kubernetes-addons/aws-for-fluentbit/locals.tf:23]: can we use the default here?

**Review by kcoleman731 — COMMENTED** [review]
Looks great!

**Review by kcoleman731 — APPROVED** [review]

**Review by askulkarni2 — APPROVED** [review]
🚢

**Pull request merged**